boards/arduino-mkr-common/Makefile.include

```diff
@@ -17,7 +21,7 @@ ifeq ($(PROGRAMMER),jlink)
 else
   # by default, we use BOSSA to flash this board and take into account the
   # preinstalled Arduino bootloader.
-  export LINKER_SCRIPT ?= $(RIOTCPU)/sam0_common/ldscripts/$(CPU_MODEL)_arduino_bootloader.ld
+  #export LINKER_SCRIPT ?= $(RIOTCPU)/sam0_common/ldscripts/$(CPU_MODEL)_arduino_bootloader.ld
```

Comment:
what is this about? Just commenting out seems not the right way to proceed here?!

Reply:
Oops, sorry, I commented it out first to test and forgot to remove it, since actually the file doesn't exist anymore in this PR.